README.md

```diff
@@ -18,4 +18,26 @@ After that you should be able to use the device under `${LOCAL_DEVICE_NAME}` jus
 ## Use inside a ROS application
 The [`ur_robot_driver`](http://wiki.ros.org/ur_robot_driver) has a [convenience script](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/scripts/tool_communication) wrapping the above `socat` call. When running the driver, the e-Series launchfiles provide a [flag](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/launch/ur10e_bringup.launch#L14) for automatically activating the tool communication. The launchfiles also allow setting up communication parameters on the robot directly.
 
+## Acknowledgment
+Developed in collaboration between:
+[<img height="60" alt="Universal Robots A/S" src="doc/resources/ur_logo.jpg">](https://www.universal-robots.com/) &nbsp; and &nbsp;
```

Comment:
New line or not, that is the question :-)

Reply:
Yes, I think the newline should be there.

Reply:
Resolved in  0144caa